tests/jballerina-integration-test/src/test/resources/configurables/testErrorProject/configPkg/tests/main_test.bal

```diff
@@ -16,7 +16,7 @@
 
 import ballerina/test;
  
-configurable map<anydata> & readonly invalidMap = ?;
+configurable map<string> & readonly invalidMap = ?;
```

Comment:
since `configurable` variables are now implicitly `readonly`, lets remove the intersection